pom.xml

```diff
@@ -1273,6 +1375,8 @@
               <spark.ui.showConsoleProgress>false</spark.ui.showConsoleProgress>
               <spark.driver.allowMultipleContexts>true</spark.driver.allowMultipleContexts>
               <spark.unsafe.exceptionOnMemoryLeak>true</spark.unsafe.exceptionOnMemoryLeak>
+              <!-- Needed by sql/hive tests. -->
+              <test.src.tables>src</test.src.tables>
```

Comment:
Ah, this is from my patch, but it should be the name of a table that is not used by the Spark tests (such as `__not_used__`). I guess it works because we don't have JUnit-based tests that trigger the relevant code paths.